main.tf

```diff
@@ -1,5 +1,5 @@
 data "aws_region" "default" {
-  current = true
+  name = "eu-west-1"
```

Comment:
@joshham1992 
we should not hardcode the name here.
if `name` is needed for the data source, use a variable